mod-invoice-storage/schemas/invoice.json

```diff
@@ -154,6 +154,14 @@
       "description": "This would keep the invoice from being feed into the batch process (Not generate a external voucher/payment) but would still move values in the system. Note: this is ideally defined by the vendor relationship and exposed for override on the invoice.",
       "type": "boolean"
     },
+    "acqUnitIds": {
+      "description": "acquisition unit ids associated with this invoice",
+      "id": "units",
```

Comment:
You could change id to `"acqUnitIds"`, so no need to change sample files

Reply:
Removed it to not confuse anyone.